testing/src/test/java/io/stargate/it/http/GraphqlTest.java

```diff
@@ -98,88 +94,106 @@ public GraphqlTest(ClusterConnectionInfo backend) {
   }
 
   @BeforeEach
-  public void setup()
-      throws InvalidSyntaxException, ExecutionException, InterruptedException, IOException {
+  public void setup(ClusterConnectionInfo cluster) throws IOException {
     keyspace = "betterbotz";
 
-    Persistence persistence = getOsgiService("io.stargate.db.Persistence", Persistence.class);
-    ClientState clientState = persistence.newClientState("");
-    QueryState queryState = persistence.newQueryState(clientState);
-    dataStore = persistence.newDataStore(queryState, null);
-    logger.info("{} {} {}", clientState, queryState, dataStore);
-
-    dataStore
-        .query()
-        .create()
-        .keyspace("betterbotz")
-        .ifNotExists()
-        .withReplication("{ 'class' : 'SimpleStrategy', 'replication_factor' : 1 }")
-        .execute();
-
-    dataStore.waitForSchemaAgreement();
-
-    dataStore
-        .query()
-        .create()
-        .table("betterbotz", "products")
-        .ifNotExists()
-        .column("id", Type.Uuid, Kind.PartitionKey)
-        .column("name", Type.Text, Kind.Clustering)
-        .column("price", Type.Decimal, Kind.Clustering)
-        .column("created", Type.Timestamp, Kind.Clustering)
-        .column("prod_name", Type.Text)
-        .column("customer_name", Type.Text)
-        .column("description", Type.Text)
-        .execute();
-
-    dataStore.waitForSchemaAgreement();
-
-    dataStore
-        .query()
-        .create()
-        .table("betterbotz", "orders")
-        .ifNotExists()
-        .column("prod_name", Type.Text, Kind.PartitionKey)
-        .column("customer_name", Type.Text, Kind.Clustering)
-        .column("id", Type.Uuid)
-        .column("prod_id", Type.Uuid)
-        .column("address", Type.Text)
-        .column("description", Type.Text)
-        .column("price", Type.Decimal)
-        .column("sell_price", Type.Decimal)
-        .execute();
-
-    dataStore.waitForSchemaAgreement();
```

Comment:
if we don't need the `dataStore.waitForSchemaAgreement();` anymore, then it is a strong reason to replace it with `CqlSession`

Reply:
This was never needed. `DataStore` automatically wait for schema agreement after DDL queries. I'm not sure why this was explicitly done.